### PR TITLE
upgrading the package versions and adding known issues

### PR DESCRIPTION
**Pull request by bhatiaPankaj**
- Upgrading the package versions to 2.0.0 along with their dependant packages
- Adding the known issues in each readme file

### CODE REVIEW

File: packages/checkbox/README.md [hunk 1]
```diff
@@ -43,3 +43,5 @@ import { Checkbox } from '@swc-uxp-wrappers/checkbox';
 ```html
 <sp-checkbox></sp-checkbox>
 ```
+## Known Issues
+- Tab navigation is not supported
```
Comment (inasirhussain) [inline packages/checkbox/README.md:47]: This is supported. Can you please confirm.

File: packages/field-group/package.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
     },
     "dependencies": {
         "@swc-uxp-internal/field-group": "npm:@spectrum-web-components/field-group@0.37.0",
-        "@swc-uxp-wrappers/help-text": "1.0.0"
+        "@swc-uxp-wrappers/help-text": "2.0.5"
```
Comment (inasirhussain) [inline packages/field-group/package.json:27]:
```suggestion
        "@swc-uxp-wrappers/help-text": "2.0.0"
```

File: packages/link/README.md [hunk 3]
```diff
@@ -43,3 +43,5 @@ import { Link } from '@swc-uxp-wrappers/link';
 ```html
 <sp-link></sp-link>
 ```
+## Known Issues
+- The double underline does not appear on hyperlinks on tabbing in Windows.
```
Comment (inasirhussain) [inline packages/link/README.md:47]: Lets just discuss this issue once.

File: packages/meter/README.md [hunk 4]
```diff
@@ -43,3 +43,7 @@ import { Meter } from '@swc-uxp-wrappers/meter';
 ```html
 <sp-meter></sp-meter>
 ```
+
+## Known Issues
+- Duplicate labels appear in React context
```
Comment (inasirhussain) [inline packages/meter/README.md:48]: It's not an issue just in React Context, but in vanilla and even on Web as well. 
Mention the workaround, to use label attribute to provide contextual name to the meter.

File: packages/number-field/README.md [hunk 5]
```diff
@@ -43,3 +43,9 @@ import { NumberField } from '@swc-uxp-wrappers/number-field';
 ```html
 <sp-number-field></sp-number-field>
 ```
+
+## Known Issues
+- Quiet and Readonly variant has black background for Windows
+- Stepper icons are trimmed with quiet variant
```
Comment (inasirhussain) [inline packages/number-field/README.md:49]:
```suggestion
- Stepper icons are trimmed with quiet variant
```
Why is this issue still there?

File: packages/switch/README.md [hunk 6]
```diff
@@ -43,3 +43,7 @@ import { Switch } from '@swc-uxp-wrappers/switch';
 ```html
 <sp-switch></sp-switch>
 ```
+
+## Known Issues
+- Tab navigation does not work
```
Comment (inasirhussain) [inline packages/switch/README.md:48]: This should work, can you confirm it?

File: packages/textfield/README.md [hunk 7]
```diff
@@ -43,3 +43,11 @@ import { Textfield } from '@swc-uxp-wrappers/textfield';
 ```html
 <sp-textfield></sp-textfield>
 ```
+## Known Issues
+- Focus ring is not curved on the corners
+- Grows API is not supported
```
Comment (inasirhussain) [inline packages/textfield/README.md:48]: Grow API is supported isnt?

File: packages/checkbox/README.md (hunk 1 above)
Comment (bhatiaPankaj) [reply packages/checkbox/README.md:47]: This is dependant on opacity support for input widget in PS which is `in-progress`

File: packages/switch/README.md (hunk 6 above)
Comment (bhatiaPankaj) [reply packages/switch/README.md:48]: This too is dependant on opacity support for input widget in PS which is in-progress

File: packages/link/README.md (hunk 3 above)
Comment (bhatiaPankaj) [reply packages/link/README.md:47]: Retained the line as per the discussion

File: packages/meter/README.md (hunk 4 above)
Comment (bhatiaPankaj) [reply packages/meter/README.md]: Added the workaround

File: packages/number-field/README.md (hunk 5 above)
Comment (bhatiaPankaj) [reply packages/number-field/README.md]: Retained the line as per the discussion

File: packages/textfield/README.md (hunk 7 above)
Comment (bhatiaPankaj) [reply packages/textfield/README.md]: Added the grows api along with known issues.

**Pull request merged**